### DISCUSSION TITLE
Language Choice

### DISCUSSION THREAD

**Discussion by moakdesigns** [AppsAgg/AppAgg#2] [Ideas]
Is there a way we can filter our search based on languages too? I see a lot of apps that aren't in my language and filtering out the languages that I don't speak would certainly help narrow down the list of available apps.

**Comment by zhdanau** [comment]
You can use the **@lng** search operator.

For example:
https://appagg.com/search/@lng%20ko
https://appagg.com/search/@lng%20es
https://appagg.com/search/@lng%20ja
& etc.
